static/includes/InstanceGPUProcedure.md

```diff
@@ -0,0 +1,10 @@
+(Optional) Configure **GPU Devices** to attach available GPU devices, enabling the instance to utilize hardware acceleration for graphics or computation tasks.
```

Comment:
same comments as mentioned above in the index file comments to show parallel contruction. 
```suggestion
(Optional) Configure GPU devices in the **GPU Devices** section to attach available GPU devices, enabling the instance to utilize hardware acceleration for graphics or computation tasks.
```